src/ImageRunModal.jsx

```diff
@@ -106,7 +106,7 @@ const EnvVar = ({ id, item, onChange, idx, removeitem, additem }) =>
         </>
     );
 
-const Volume = ({ id, item, onChange, idx, removeitem, additem }) =>
+const Volume = ({ id, item, onChange, idx, removeitem, additem, options }) =>
```

Comment:
I'd rather leave out the `options` indirection and specify them as direct properties. More robust and easier to check with propTypes and such if they are flat.

That's not a veto though, if you prefer that it's fine.

Reply:
The thing is that this is wrapped with `DynamicListForm` which is used for different components. Adding random options for whichever component we would need them would make it rather messy, thus I decided  to go with general `options` so any component can specify what they need without making this wrapper one big mess.

Reply:
Makes sense, thanks!